cmake/scripts/common/AddonHelpers.cmake

```diff
@@ -182,6 +182,24 @@ macro (build_addon target prefix libs)
   if(EXISTS ${PROJECT_SOURCE_DIR}/${target}/addon.xml.in)
     set_property(DIRECTORY APPEND PROPERTY CMAKE_CONFIGURE_DEPENDS ${PROJECT_SOURCE_DIR}/${target}/addon.xml.in)
     set(PLATFORM ${CORE_SYSTEM_NAME})
+    set(PLATFORM_TAG ${CORE_SYSTEM_NAME})
+
+    if(CORE_SYSTEM_NAME STREQUAL windows)
+      include(CheckSymbolExists)
+      check_symbol_exists(_X86_ "Windows.h" _X86_)
+      check_symbol_exists(_AMD64_ "Windows.h" _AMD64_)
```

Comment:
shouldn't this be known from kodiconfig.cmake? if not, it'd be worth adding there

Reply:
Seems a bit hacky. And we have to split the arch for android as well. Linux also, if in-scope.
We probably need a more structured way to get the arch/cpu here...

Reply:
This was only hacked in to show it works, needs to be done properly for Windows, Android, iOS and OS X.
For Linux we still use the ppa, and distros like LibreELEC should define their own naming if they want a binary addon repo.

Reply:
Ok. So, If I got you right, we might as well do a mass rename in jenkins, which is already per arch, can we?